docs/source/getting-started/first-feature.rst

```diff
@@ -180,13 +180,13 @@ Let's give the new functionality a spin.
 Running the New Feature
 =======================
 
-We need to terminate the previous ``./daml-start.sh`` process and run it again, as we need to have a Sandbox instance with a DAR file containing the new feature. As a reminder, by running ``./daml-start.sh`` again we will 
+We need to terminate the previous ``daml start --start-navigator=no`` process and run it again, as we need to have a Sandbox instance with a DAR file containing the new feature. As a reminder, by running ``daml start --start-navigator=no`` again we will
```

Comment:
I don't think we need to write the `--start-navigator=no` argument when just referring to the process that's running or that you want to kill. I think just referring to the `daml start` process is enough, though of course you need to give the full command when asking the user to run it. Maybe this point will be moot if we change the defaults.